tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -546,3 +547,110 @@ def test_positive_virt_card(session, target_sat, module_location, module_org, vm
         target_sat.api.Host(
             id=target_sat.api.Host().search(query={'search': f'name={host_name}'})[0].id
         ).delete()
+
+
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize(
+    'setting_update',
+    ['remote_execution_connect_by_ip=True', 'destroy_vm_on_host_delete=True'],
+    indirect=True,
+)
+@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
+@pytest.mark.parametrize('provision_method', ['build'])
+@pytest.mark.rhel_ver_match('[8]')
```

Comment:
Why is this, what fixture creates RHEL hosts?

Reply:
@lhellebr It is needed by internal fixture `module_provisioning_rhel_content` for syncing the content repos based on the host we are provisioning